### ISSUE TITLE
Create Property Set messed up

### ISSUE THREAD

**Issue by BobRay** [modxbot/migrate#11087]
bobray created Redmine issue ID 11087

Steps to reproduce 1. Edit the description of an existing default property - save.
2. Click on Add property set - give name - click save in the popup
3. Property set has the name of the previously edited default property.
4. Repeat 2-3
5. Property set name shows as a number
6. Subsequently created prop sets are saved, but don't show in dropdown on the element's properties page. The first property set has the wrong name in the database, but otherwise, both property set tables in the DB appear to be correct. Prop sets are created and linked to the correct element. If you edit another element, clear the cache, and come back, the new property sets still don't show (except the one named for the default property).

**Comment by splittingred** [comment]
splittingred submitted:

Ok, so I can get #3 to happen, but not #5 or #6, at all. #3 is pretty trivial. I cant get the "missing property sets" to happen, nor can I get any of the other bugs to happen. I followed your steps to the T. Can you try again?

**Comment by BobRay** [comment]
bobray submitted:

It's completely reliable for me. In fact none of my newer property sets show at all in the drop-down on the properties grid whether or not I do step 1 and no matter what kind of element I'm working on. As soon as I save a new property set for any element, the property set's ID number shows in the drop down box. After I leave the element and come back, the new property set doesn't show at all in the drop-down. Property sets I created a while ago do show, but none I create now do. The new property sets are in the modx_property_sets table and they show in Tools | Property sets, but I can't get them to show in the drop-down on the element's properties grid. I don't see how this is even possible but it's absolutely reliable. I've tried clearing the cache and the browser cache/cookies, logging out, restarting the browser, doing an upgrade install &#8211; nada. They just don't show. Maybe it's time for a fresh checkout?
